server/graphql/v2/object/Update.js

```diff
@@ -0,0 +1,118 @@
+import {
+  GraphQLBoolean,
+  GraphQLEnumType,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from 'graphql';
+import { GraphQLDateTime } from 'graphql-iso-date';
+
+import { stripHTML } from '../../../lib/sanitize-html';
+import { DateString } from '../../v1/types';
+import { getIdEncodeResolver, IDENTIFIER_TYPES } from '../identifiers';
+import { Account } from '../interface/Account';
+
+import { Tier } from './Tier';
+
+const UpdateAudienceTypeEnum = new GraphQLEnumType({
```

Comment:
We should move that to the `enum` folder, it's meant to be used by the `createMutation` too when we'll have it on v2